examples/submit_job.py

```diff
@@ -0,0 +1,29 @@
+"""Example of how to submit a Dalton job using Python HTTP requests."""
+import requests
+import os
+
+DALTON_URL = os.getenv("DALTON_URL", "localhost")
+
+# this file needs to include the sensor configuration in yaml format. Make sure that the files are in the proper directory.
+conf = open("engine_conf.yaml", "r") 
+configuration = conf.read()
+
+# this file needs to include any custom rules you want to test
+cust = open("custom_rules.rules", "r") 
+custom_rules = cust.read()
+
+# in this example we use suricata as sensor
+data = {
+  "sensor_tech": "suricata/6.0.4/suricata.yaml",
+  "optionProdRuleset": "prod",
+  "prod_ruleset": "/opt/dalton/rulesets/suricata/suricata.rules",
```

Comment:
You can dynamically pull lists of rulesets by  sensor too (https://github.com/secureworks/dalton/blob/master/app/dalton.py#L213):

DALTON_URL/dalton/controller_api/get-prod-rulesets/suricata
DALTON_URL/dalton/controller_api/get-prod-rulesets/snort